Assets/Sprites/invis.asset

```diff
@@ -0,0 +1,36 @@
+%YAML 1.1
```

Comment:
It seems like `Square.png` and `invis.asset` are intended to be used for invisible hitboxes. I might simplify how invisible hitboxes are set up in the level.